polkadot/xcm/procedural/tests/builder_pattern.rs

```diff
@@ -22,11 +22,11 @@ use xcm::latest::prelude::*;
 #[test]
 fn builder_pattern_works() {
 	let asset: Asset = (Here, 100u128).into();
-	let beneficiary: Location = AccountId32 { id: [0u8; 32], network: None }.into();
-	let message: Xcm<()> = Xcm::builder()
-		.receive_teleported_asset(asset.clone().into())
+	let beneficiary: Location = [0u8; 32].into();
+	let message: Xcm<()> = Xcm::builder_unsafe()
```

Comment:
Why did we change it to `builder_unsafe`?

Reply:
Was testing some stuff and didn't realize 😅, changing it back